cpp/vcpkg.json

```diff
@@ -16,11 +16,16 @@
     },
     "benchmark",
     "boost-cmake",
+    "boost-beast",
```

Comment:
Could you keep this list in alphabetical order?

Reply:
Fixed